src/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -11,6 +11,7 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using SysTx = System.Transactions;
```

Comment:
Why do you need this alias here? Are there duplicate Transaction classes in this scope?